docs/UserGuide.md

```diff
@@ -206,25 +206,36 @@ _Details coming soon ..._
+| **Create a new event**                        | `ecreate n/EVENT_NAME r/ROLES_NEEDED… d/DATE_AND_TIME l/LOCATION dsc/DESCRIPTION [m/MATERIALS_AND_LOGISTICS_NEEDED]... [b/BUDGET]` <br> e.g., `ecreate n/Fundraising at Orchard r/logistics leader d/23-09-2023 1500 dsc/station at locations to ask for donations m/50 tin cans b/50`                                 |
+| **List all events**                           | `elist`                                                                                                                                                                                                                                                                                                                |
+| **Read an individual event**                  | `eshow EVENT_ID` <br> e.g., `eshow 8`                                                                                                                                                                                                                                                                                  |
+| **Delete an event**                           | `edelete EVENT_ID` <br> e.g., `edelete 3`                                                                                                                                                                                                                                                                              |
+| **Create a new volunteer profile**            | `vcreate vn/VOLUNTEER_NAME hp/PHONE_NUMBER e/EMAIL`<br> e.g.,`vcreate vn/John Lim hp/81234567 e/john123@gmail.com`                                                                                                                                                                                                     |
+| **List all volunteers**                       | `vlist`                                                                                                                                                                                                                                                                                                                |
+| **Delete a volunteer profile**                | `vdelete VOLUNTEER_ID` <br> e.g., `vdelete 4`                                                                                                                                                                                                                                                                          |
+| **Add a volunteer to an event**               | `vadd vn/VOLUNTEER hp/PHONE_NUMBER e/EMAIL /to eid/EVENT_ID`<br> e.g., `vadd vn/John hp/81234567 e/john123@gmail.com /to eid/3` <br> Alternatively, `vadd vn/VOLUNTEER_NAME hp/PHONE_NUMBER e/EMAIL /to en/EVENT_NAME` <br> e.g., `vadd vn/John hp/81234567 e/john123@gmail.com /to en/Fundraising for needy families` |
```

Comment:
i think the command format has changed, can see max's pull request

Reply:
oki